### ISSUE TITLE
Duplicated primary keys in wazuh_db

### ISSUE THREAD

**Issue by Skeptor** [wazuh/wazuh#1258]
While running the Wazuh Manager 3.7 there is an error with wazuh_db:

```
2018/09/07 16:54:57 wazuh-db: ERROR: at wdb_netproto_insert(): sqlite3_step(): UNIQUE constraint failed: sys_netproto.id, sys_netproto.type
2018/09/07 16:54:57 wazuh-db: ERROR: Unable to update 'sys_netproto' table for agent '000'
2018/09/07 16:54:57 ossec-analysisd: ERROR: at sc_send_db(): received: 'err Cannot save netproto information.'
2018/09/07 16:54:57 ossec-analysisd: ERROR: Unable to send netinfo message to Wazuh DB.
```
It tries to insert a new entry but there is a duplicated primary key.
This is the ifconfig print from my computer:
```
docker0   Link encap:Ethernet  direcciónHW 02:42:bd:24:51:2b  
          Direc. inet:172.17.0.1  Difus.:172.17.255.255  Másc:255.255.0.0
          ACTIVO DIFUSIÓN MULTICAST  MTU:1500  Métrica:1
          Paquetes RX:0 errores:0 perdidos:0 overruns:0 frame:0
          Paquetes TX:0 errores:0 perdidos:0 overruns:0 carrier:0
          colisiones:0 long.colaTX:0 
          Bytes RX:0 (0.0 B)  TX bytes:0 (0.0 B)

enp2s0    Link encap:Ethernet  direcciónHW 68:f7:28:84:b1:0d  
          Direc. inet:192.168.1.54  Difus.:192.168.1.255  Másc:255.255.255.0
          Dirección inet6: fe80::c9e5:7a7a:b735:eb04/64 Alcance:Enlace
          ACTIVO DIFUSIÓN FUNCIONANDO MULTICAST  MTU:1500  Métrica:1
          Paquetes RX:77492 errores:0 perdidos:0 overruns:0 frame:0
          Paquetes TX:29200 errores:0 perdidos:0 overruns:0 carrier:0
          colisiones:0 long.colaTX:1000 
          Bytes RX:105863445 (105.8 MB)  TX bytes:2627826 (2.6 MB)

lo        Link encap:Bucle local  
          Direc. inet:127.0.0.1  Másc:255.0.0.0
          Dirección inet6: ::1/128 Alcance:Anfitrión
          ACTIVO BUCLE FUNCIONANDO  MTU:65536  Métrica:1
          Paquetes RX:32423 errores:0 perdidos:0 overruns:0 frame:0
          Paquetes TX:32423 errores:0 perdidos:0 overruns:0 carrier:0
          colisiones:0 long.colaTX:1000 
          Bytes RX:98764879 (98.7 MB)  TX bytes:98764879 (98.7 MB)

vboxnet0  Link encap:Ethernet  direcciónHW 0a:00:27:00:00:00  
          Direc. inet:10.0.0.1  Difus.:10.0.0.255  Másc:255.255.255.0
          Dirección inet6: fe80::800:27ff:fe00:0/64 Alcance:Enlace
          ACTIVO DIFUSIÓN MULTICAST  MTU:1500  Métrica:1
          Paquetes RX:0 errores:0 perdidos:0 overruns:0 frame:0
          Paquetes TX:120 errores:0 perdidos:0 overruns:0 carrier:0
          colisiones:0 long.colaTX:1000 
          Bytes RX:0 (0.0 B)  TX bytes:13935 (13.9 KB)

wlp3s0    Link encap:Ethernet  direcciónHW d0:7e:35:e7:b0:76  
          Direc. inet:192.168.2.242  Difus.:192.168.2.255  Másc:255.255.255.0
          Dirección inet6: fe80::8d61:3a9d:15e2:f855/64 Alcance:Enlace
          ACTIVO DIFUSIÓN FUNCIONANDO MULTICAST  MTU:1500  Métrica:1
          Paquetes RX:904 errores:0 perdidos:0 overruns:0 frame:0
          Paquetes TX:377 errores:0 perdidos:0 overruns:0 carrier:0
          colisiones:0 long.colaTX:1000 
          Bytes RX:125305 (125.3 KB)  TX bytes:60529 (60.5 KB)
```

**Comment by Skeptor** [comment]
After some research the problem was found in the multi-thread scanner. 
The messages are received without order so Wazuh-DB assign differents IDs to the protocols of the same network interface.
For example, if you have two interfaces: Iface1 and Iface2.
The Wazuh-DB must contain:
``1 - ipv4 from Iface1
| 1 - ipv6 from Iface1
| 2 - ipv4 from Iface2
| 2 - ipv6 from Iface2``
But in the wrong order it could try to insert:
``1 - ipv4 from Iface1
| 1 - ipv4 from Iface2``
And there the DB detects a duplicate Primary key (1,ipv4) == (1,ipv4)

**Comment by chemamartinez** [comment]
Solved at #1315
